extensions/qute/runtime/src/main/java/io/quarkus/qute/runtime/extensions/StringTemplateExtensions.java

```diff
@@ -67,4 +67,9 @@ static String fmt(String ignoredPropertyName, Locale locale, String format, Obje
         return String.format(locale, format, args);
     }
 
+    @TemplateExtension(matchName = "+")
+    static String plus(String str, Object val) {
+        return str + val;
```

Comment:
Any null check here? Or the idea is to print `null` when any of these objects are null?

Reply:
`str` will never be `null`. If `val` is `null` then just append "null".